tvm_client/src/net/network_params.rs

```diff
@@ -45,7 +45,7 @@ pub async fn get_signature_id(
     }
 }
 
-pub(crate) fn offline_config() -> (BlockchainConfig, i32) {
+pub(crate) fn _offline_config() -> (BlockchainConfig, i32) {
```

Comment:
As I remember this function was added to work with networks without masterchain, does it works with AckiNacki net?